### PR TITLE
RIGs now splint broken limbs correctly

### PR DESCRIPTION
**Pull request by Geevies**
Adds to fractured organ process, if the user's wearing a RIG and it's deployed over that part, it'll splint that part.

### CODE REVIEW

File: html/changelogs/geeves-organDablage.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Added a fractured var to external organs."
```
Comment (Geevies) [inline html/changelogs/geeves-organDablage.yml]:
```suggestion
```

[the remark this answers is not in the document]
Comment (Geevies) [reply html/changelogs/geeves-organDablage.yml]: fair 'nuff

File: code/modules/organs/organ_external.dm [hunk 2]
```diff
@@ -491,6 +492,17 @@ This function completely restores a damaged organ to perfect condition.
 		if(config.bones_can_break && !(status & ORGAN_ROBOT) && !(status & ORGAN_PLANT) && brute_dam > min_broken_damage * config.organ_health_multiplier)
 			src.fracture()
 
+		if(fractured)
```
Comment (NonQueueingMatt) [inline code/modules/organs/organ_external.dm]: Why not just check the ORGAN_BROKEN flag?

Comment (Geevies) [reply code/modules/organs/organ_external.dm]: species that don't have bones have weird ORGAN_BROKEN interactions, from what i know, so having a fractured flag checks specifically whether the fracture proc has fired; it's pretty much to make it foolproof

**Comment by skull132** [comment]
Adding another variable carries the issue of confusion. There's status, which is meant to indicate the state of the organ, but now there's also another variable to check? y tho.

Simply add an `ORGAN_FRACTURED` flag that's used instead. Make it be exclusive to `ORAN_BROKEN` and voila. Tho you'll have to retool every single ORGAN_BROKEN check to check for _both_. This can be done by also defining `#define ORGAN_BROKEN_OR_FRACTURED (ORAGN_BROKEN|ORGAN_FRACTURED)`.

**Comment by Geevies** [comment]
After looking at the code some more, it seems "ORGAN_BROKEN" is actually only used for broken bones. Whoda thunk?

**Pull request merged**